experimental/sql/src/ibis_frontend.py

```diff
@@ -72,7 +72,11 @@ def visit_schema(schema: ibis.expr.schema.Schema) -> Region:
 def visit_ibis_expr_list(l: List[ibis.expr.types.Expr]) -> Region:
   ops = []
   for op in l:
-    ops.append(visit(op))
+    if isinstance(op.op(), ibis.expr.operations.logical.And):
```

Comment:
What if there are several nested `And`s?

Similar question here as before: this could also be a simplification/normalization pass on the ibis dialect -- or on a later one. Note that there might be slightly more complicated cases where it is possible to detect a conjunction, for example, by applying a distributive equivalence first (`ab | bc` --> `b & (a | c)`). Any thoughts?

Reply:
This is something that we do not handle atm. It is also not something actually needed for the TPC-H queries, that's why I made the decision to ignore it.